### PR TITLE
Changes to build.gradle and code

### PR DESCRIPTION
**Pull request by DimosTzanakos**
Added manifest for the .jar files in build.gradle.
Did some basic changes to the code.

### CODE REVIEW

**Comment by DimosTzanakos** [comment]
what should i remove from build/?

**Comment by namezys** [comment]
anything. we have to not have directory "build" in repo

File: src/main/java/ViewHotelRoomsFrame.java [hunk 1]
```diff
@@ -25,10 +28,18 @@ public ViewHotelRoomsFrame(Hotel h, Guest g) {
...
         validate();
 
-        image2 = new ImageIcon(getClass().getResource("rm2.png"));
+        image2 = new ImageIcon(getClass().getResource("room2.png"));
         label2 = new JLabel(image2);
         add(label2);
-
+/*
+	image3 = new ImageIcon(getClass().getResource("Images/room3.png"));
+        label3 = new JLabel(image3);
```
Comment (namezys) [inline src/main/java/ViewHotelRoomsFrame.java]: Maybe you can use test message instead of image
I think it will be like `new JLabel("Room 3:")

**Comment by DimosTzanakos** [comment]
I commented the two lines out. So now it does not give any errors.
Its just that the new window that opens by selecting the button View/Change Reservation opens empty, with no images in it.
Maybe we can accept this pull request for now and check the code later...

**Pull request merged**